### PR TITLE
Setup react

### LINKED ISSUE #9: Setup React to work with express

### PR DESCRIPTION
**Pull request by lustoykov**
Close #9

### CODE REVIEW

File: express/app.js [hunk 1]
```diff
@@ -29,6 +29,8 @@ app.use(require('node-sass-middleware')({
   sourceMap: true,
 }));
 app.use(express.static(path.join(__dirname, 'public')));
+// Make React Resources available
+app.use('/react', express.static(path.join(__dirname, '/../react/dist')));
```
Comment (martomi) [inline express/app.js]: Ping #15 - This directory is not copied to docker. Either webpack should put the build in express/react or this folder needs to be copied to docker.

Comment (lustoykov) [reply express/app.js]: changed react build path

**Review by martomi — APPROVED** [review]
Looks good! 👍

**Review by chaoran-chen — APPROVED** [review]
Looks also good for me! Maybe, it would be better if we can split the React-core-code and our components into two files, so that the react-core-code can be cached and doesn't need to be loaded with every app-bundle. But that's not so important, we can think about it later.

**Comment by sacdallago** [comment]
Delete merged branches!!!!!!!
|| 
|| 
||  
▼

**Pull request merged**